server/asset/eth/eth.go

```diff
@@ -317,9 +319,10 @@ func (eth *Backend) Redemption(redeemCoinID, contractCoinID []byte) (asset.Coin,
 	return cnr, nil
 }
 
-// FundingCoin is an unspent output.
-func (eth *Backend) FundingCoin(ctx context.Context, coinID []byte, redeemScript []byte) (asset.FundingCoin, error) {
-	return nil, notImplementedErr
+// FundingCoin is an unspent amount on the blockchain. It points to an account
+// that should maintain at least a certain value.
+func (eth *Backend) FundingCoin(ctx context.Context, coinID []byte, _ []byte) (asset.FundingCoin, error) {
+	return newAmountCoin(eth, coinID)
 }
```

Comment:
I don't think we need to worry about `redeemScript`, but, It could be used to determine the spend size. I wonder if the SwapCoinID should include data on whether that coin represents an initiation or a redemption. That data, included here, would allow us to decide the spend size.